bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoUnitSystem.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.internal;
+
+import java.math.BigDecimal;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * @author Rob Nielsen
+ * @since 1.8.0
+ *
+ *        This enum holds all the different unit systems for the Netatmo binding,
+ *        along with conversion methods
+ */
+public enum NetatmoUnitSystem {
+	M("Metric"), US("US");
+	
+	public static final NetatmoUnitSystem DEFAULT_UNIT_SYSTEM = NetatmoUnitSystem.M;
+
+	String unitSystem;
+
+	private NetatmoUnitSystem(String unitSystem) {
+		this.unitSystem = unitSystem;
+	}
+
+	public String getunitSystem() {
+		return unitSystem;
+	}
+
+	public static NetatmoUnitSystem fromString(String unitSystem) {
+		if (!StringUtils.isEmpty(unitSystem)) {
+			for (NetatmoUnitSystem unitSystemType : NetatmoUnitSystem.values()) {
+				if (unitSystemType.toString().equalsIgnoreCase(unitSystem)) {
+					return unitSystemType;
+				}
+			}
+		}
+		throw new IllegalArgumentException("Invalid unitSystem: " + unitSystem);
+	}
+	
+	public double convertAltitude(double value) {
+		if (this == DEFAULT_UNIT_SYSTEM) {
+			return value;
+		}
+		
+		return value * 3.2808399;
+	}
+	
+	public BigDecimal convertRain(BigDecimal value) {
+		if (this == DEFAULT_UNIT_SYSTEM) {
+			return value;
+		}
+		
+		return value.multiply(new BigDecimal(0.0393700787));
+	}
+	
+	public BigDecimal convertTemp(BigDecimal value) {
+		if (this == DEFAULT_UNIT_SYSTEM) {
+			return value;
+		}
+		
+		return value.multiply(new BigDecimal(9)).divide(new BigDecimal(5)).add(new BigDecimal(32));
```

Comment:
Better to have private static final BigDecimal values NINE, FIVE, THIRTY_TWO to avoid creating new objects on each conversion.